task-2/response-helper.js

```diff
@@ -0,0 +1,42 @@
+const RESPONSE_HEADER_CONTENT_TYPE = 'content-type';
```

Comment:
good, but next time consider this https://nodejs.org/api/http.html#http_http_status_codes